src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -383,18 +383,35 @@ export default class AircraftController {
     }
 
     /**
+     * Finds an aircraft by its callsign
+     *
      * @method findAircraftByCallsign
      * @param  {string} [callsign='']
      * @return {AircraftModel|null}
      */
     findAircraftByCallsign(callsign = '') {
-        if (callsign === '') {
+        if (!callsign) {
```

Comment:
I like this better, but perhaps we should change the default assignment to `null` then instead of an empty string?

Reply:
Good point

Reply:
done